go-controller/pkg/ovn/egressip.go

```diff
@@ -1587,17 +1587,17 @@ func (oc *Controller) setNodeEgressReachable(nodeName string, isReachable bool)
 	}
 }
 
-func (oc *Controller) addEgressNode(egressNode *kapi.Node) error {
+func (oc *Controller) addEgressNode(egressNode string) error {
```

Comment:
NIT: wdyt we rename this to egressNodeName or nodeName?

Reply:
since the function is called `addEgressNode` could we just go with `nodeName`?

Reply:
sure yes,